chemicaltoolbox/rdkit/sdf_to_tab.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+import argparse
+import pandas as pd
+from rdkit import Chem
+
+def sdf_to_tab(vars):
+    mols = Chem.SDMolSupplier(vars.inp, sanitize=False)
+    df = pd.DataFrame()  # for output
+
+    for mol in mols:
+        if mol:
+            d = mol.GetPropsAsDict()
+            # filter dict for desired props
+            if vars.props.strip() == '':  # none specified, return all
+                d = {prop: val for (prop, val) in d.items() if not any(x in str(val) for x in ['\n', '\t'])}  # remove items containing newlines or tabs
+            else:
+                d = {prop: val for (prop, val) in d.items() if prop in vars.props.replace(' ', '').split(',')}  # remove items not requested via CLI
+            d['Name'] = mol.GetProp('_Name')
```

Comment:
This looks dangerous - many SDFs don't have a name field (first line). In those cases I think this will throw and error as the key is not in the dict? Or if is is then it will be None or empty string. I would only include the Name field in the output if it is really defined in the first line of the SDF record.
Also, I think it would be useful to always include the record index as an output field?

Reply:
It doesn't throw an error, it just returns an empty string. So I think this is safe enough.

Adding columns for record index and SMILES is a good idea, I'll add those.